### PR TITLE
filter by county

### LINKED ISSUE #124: add county filter
county data should be available soon although the completion date for the data survey is unknown at this time. 

add a filter to narrow selections by county adjusting the map as necessary which will likely require getting the geographic centers of each county

### PR DESCRIPTION
**Pull request by nikolaiwarner**
fixes #124 

You can check this out live here, if you want:
https://gethelplex-nikolaiwarner.c9users.io

Also, I fiddled with the layout and style a bit. I hope you dont mind. But this sets us up to do cooler things in the future.

### CODE REVIEW

**Comment by chaserx** [comment]
niiiice!

**Comment by chaserx** [comment]
screen shot 2016-08-25 at 3 19 28 pm

**Comment by chaserx** [comment]
travis seems to be upset about something

File: src/ui/map.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ define(function(require, exports, module) {
 
     this.configureMap = function(ev, config) {
       this.trigger('mapStarted', {});
-      // if list or facets are emabled, give the map less space
```
Comment (eeeschwartz) [inline src/ui/map.js:39]: "emabled" has its own charm but ok

**Comment by eeeschwartz** [comment]
+1 looks great

**Comment by chaserx** [comment]
+1

File: src/ui/map.js (hunk 1 above)
Comment (nikolaiwarner) [reply src/ui/map.js:39]: yeah. i pondered this one for a while. what right do i have to cleanse history with my contemporary interpretation of English language?

**Comment by nikolaiwarner** [comment]
thanks!! <3

**Pull request merged**